site/app/templates/admin/users/GraderList.twig

```diff
@@ -1,55 +1,52 @@
 <div class="content">
-    <div style="float: right; margin-bottom: 20px;">
-        <a href="javascript:newDownloadForm()" class="btn btn-primary">Download Graders</a>
-        <a href="javascript:newGraderListForm()" class="btn btn-primary">Upload Grader List</a>
-        <a href="javascript:newGraderForm()" class="btn btn-primary">New Grader</a>
-		<a href="javascript:editRegistrationSectionsForm()" class="btn btn-primary">Assign Registration Sections for Grading</a>
-    </div>
-    <h1>Manage Graders</h1>
+    <header>
+        <h1>Manage Graders</h1>
+        <div class="btn-wrapper">
+            <a href="javascript:newDownloadForm()" class="btn btn-primary">Download Graders</a>
+            <a href="javascript:newGraderListForm()" class="btn btn-primary">Upload Grader List</a>
+            <a href="javascript:newGraderForm()" class="btn btn-primary">New Grader</a>
+            <a href="javascript:editRegistrationSectionsForm()" class="btn btn-primary">Assign Registration Sections for Grading</a>
+        </div>
+    </header>
     {# This is a data table #}
-    <table class="table table-striped table-bordered persist-area">
-        <caption />
-        <thead class="persist-thead">
-            <tr>
-                <th width="4%"></th>
-                <th width="20%" style="text-align: left">User ID</th>
-                <th width="30%" colspan="2">Name</th>
-                <th width="23%">User Group</th>
-                <th width="20%">Registration Sections</th>
-                <th width="3%"></th>
-            </tr>
-        </thead>
-        <tbody>
-        {% if graders[1]|length + graders[2]|length + graders[3]|length > 0 %}
+    {% if graders[1]|length + graders[2]|length + graders[3]|length > 0 %}
+        <table id="grader-table" class="table table-striped mobile-table directory-table">
+            <caption>Graders For Class</caption>
```

Comment:
I don't really like the title, maybe something like "Course Graders"?  I can be overruled though if you like the current title.